src/components/App/index.tsx

```diff
@@ -25,18 +26,26 @@ export default function App() {
     <ThemeProvider theme={darkTheme}>
       <CssBaseline />
       <Header />
-      <Switch>
-        <Route exact path="/" component={Home} />
-        <Route path="/brokenimages" component={BrokenImages} />
-        <Route path="/checkboxes" component={Checkboxes} />
-        <Route path="/dropdown" component={Dropdown} />
-        <Route path="/datetime" component={DateTime} />
-        <Route path="/sliders" component={Sliders} />
-        <Route path="/switches" component={Switches} />
-        <Route path="/autocomplete" component={Autocomplete} />
-        <Route path="/table" component={Table} />
-        <Route path="*" component={PageNotFound} />
-      </Switch>
+      <Grid
+        container
+        alignItems="center"
+        justifyContent="center"
+        direction="column"
+        height={"100%"}
+      >
```

Comment:
This is part of what was in Custom Container, now in App file. This Grid will contain whatever is rendered in it.